dev/build-fleet

```diff
@@ -18,6 +18,14 @@ fi
 
 export GOOS=linux
 
+# The name of the container image created here is a potential source of conflict
+# when fleet is set up simultaneously, as several scripts are calling
```

Comment:
This `simultaneously` alone may leave the reader wondering what simultaneity we're talking about here.
```suggestion
# when fleet is set up simultaneously in multiple sets (ie 1 upstream +
# 0 to n downstream) of clusters, as several scripts are calling
```